src/txdb.h

```diff
@@ -48,7 +48,15 @@ class CCoinsViewDB final : public CCoinsView
 protected:
     CDBWrapper db;
 public:
-    explicit CCoinsViewDB(size_t nCacheSize, bool fMemory = false, bool fWipe = false);
+    /**
+     * @param[in] ldb_path    Location in the filesystem where leveldb data will be stored.
+     */
+    explicit CCoinsViewDB(
+        fs::path ldb_path,
+        size_t nCacheSize,
+        bool fMemory = false,
```

Comment:
In commit "refactor: have CCoins* data managed under CChainState" (eed4e62ad6b162ff0d3e2b87eb8762a689678bca)

Can we get rid of these default values? It seems like they are not actually used anymore, and default booleans params  in general tend to make code trickier to read and mask bugs.